dev/ts/main.ts

```diff
@@ -105,6 +106,21 @@ export function getElementsByIds(bpmnId: string | string[]): BpmnElement[] {
   return bpmnVisualization.bpmnElementsRegistry.getElementsByIds(bpmnId);
 }
 
+function getParentElement(id: string): mxCell {
```

Comment:
**thought:** We should provide the parent as part of the Registry API (in the future) to not depend on mxGraph code here, see #929